nephos/fabric/crypto.py

```diff
@@ -374,16 +375,18 @@ def setup_id(opts, msp_name, release, id_type):
 
 
 # TODO: Rename to mention identities.
-def setup_nodes(opts, node_type):
+def setup_nodes(opts):
     """Setup identities for nodes.
 
     Args:
         opts (dict): Nephos options dict.
-        node_type (str): Type of node.
     """
-    nodes = opts[node_type + "s"]
-    for release in nodes["names"]:
-        setup_id(opts, nodes["msp"], release, node_type)
+    for msp in get_msps(opts=opts):
+        for peer in get_peers(opts=opts, msp=msp):
+            setup_id(opts, msp, peer, "peer")
+
+    for orderer in get_orderers(opts=opts, msp="AlphaMSP"):
```

Comment:
I would do this for all MSPs, but we will only obtain orderers from MSPs that have an orderers section. This way we don't hardcode the Orderers to a specific "AlphaMSP"